packages/wallets/src/features/cashier/screens/WalletDepositScreens/WalletDepositFiat/WalletDepositFiat.scss

```diff
@@ -1,7 +1,11 @@
 .wallets-deposit-fiat {
     &__iframe {
-        width: 50%;
-        height: 65vh;
+        width: 78.4%; // as given in design
```

Comment:
I think this should be a fixed value in px. The iframe might look stretched in ultra wide screens.